srcpkgs/wired-notify/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'wired-notify'
+pkgname=wired-notify
+version=0.10.2
+revision=1
+archs="i686 x86_64"
+build_style=cargo
+hostmakedepends="pkg-config"
+makedepends="rust cargo pango-devel libXScrnSaver-devel libglib-devel"
```

Comment:
`rust` and `cargo` would go in `hostmakedepends`, but they are implied by `build_style=cargo`